src/mame/aristocrat/aristmk7.cpp

```diff
@@ -0,0 +1,89 @@
+// license:BSD-3-Clause
+// copyright-holders:
+/**************************************************************************************************
+
+Aristocrat MK-7
+
+Custom PC-based motherboard labeled MK7 CARRIER BOARD:
+- Celeron D 1.6 GHz;
+- Intel 82945GM (northbridge + Calistoga GPU);
+- Intel NH82801GBM (SL8YB);
+- Intel 82573L;
+- 2x 512MB SODIMMs;
+- CF slot, dumps are in ext2 format;
+- 2 SATA slots, apparently some games uses these instead;
+- SIM card slot (dongle, each game is tied with a SIM, presumably have files inside)
+- USB port (2.0 x1?), 2x 8-pin socketed chips (?)
+- A serial port near JTAG labels, a 2x 7-seg LEDs and x2 Panasonic BR-2/3A batteries labeled 
+  BT1 / BT2;
+- U57 chip near the batteries (RTC?);
+- XMULTIPLE XMH-TRJG 1730AONL ethernet port;
+- PCIe 2.0 x16 + x1 slot;
+- Altera EPM570 CPLD, Altera Cyclone II;
+- 3x onboard buttons labeled 522 T (L25, L26, L32);
+- (unconfirmed) Viridian video card a.k.a. nVidia GeForce GT430 (GF108)
```

Comment:
I tried my best sourcing this from the photo but several parts are unreadable (particularly the chip markings, specifically the existence of a Super I/O or RTC).